packages/devkit/src/utils/names.spec.ts

```diff
@@ -33,4 +33,15 @@ describe('names', () => {
     expect(names(' foo bar').fileName).toEqual('-foo-bar');
     expect(names('_foo_bar').fileName).toEqual('_foo-bar');
   });
+
+  it('should support constant names', () => {
+    expect(names('foo-bar').constantName).toEqual('FOO_BAR');
+    expect(names('foo_bar').constantName).toEqual('FOO_BAR');
+    expect(names('FooBar').constantName).toEqual('FOO_BAR');
+    expect(names('[fooBar]').constantName).toEqual('FOO_BAR');
+    expect(names('[...fooBar]').constantName).toEqual('FOO_BAR');
+    expect(names('foo-@bar').constantName).toEqual('FOO_BAR');
+    expect(names(' foo bar').constantName).toEqual('FOO_BAR');
+    expect(names('_foo_bar').constantName).toEqual('FOO_BAR');
```

Comment:
These expectations should probably match more closely, the ones for file names and class names.